tests/data/test_dataloader.py

```diff
@@ -547,11 +572,6 @@ def test_finetuning_dataloader_streaming(tmp_path: pathlib.Path):
 
     cfg = om.create(cfg)
 
-    tokenizer = build_tokenizer(
-        tokenizer_name='gpt2',
-        tokenizer_kwargs={'model_max_length': max_seq_len},
-    )
-
     _ = build_finetuning_dataloader(cfg, tokenizer, 4)
```

Comment:
should probably also make this test a little more thorough and test that the first batch in the dataloader looks how you expect